README.md

```diff
@@ -17,8 +17,123 @@ value from data whilst ensuring it is properly governed.
 
 # Postgres Connector
 
-This project contains the Postgres connector for Egeria.
+This project contains the Postgres Database Integration Daemon Connector for Egeria.
+ 
+
+#Introduction
+
+
+The Postgres Database Integration Daemon Connector is an example of an Egeria Integration Daemon connector, an integration framework which is described in detail [here.](https://egeria.odpi.org/open-metadata-implementation/admin-services/docs/concepts/integration-daemon.html#:~:text=An%20Integration%20Daemon%20is%20an,Access%20Point%20or%20Metadata%20Server.)
+To support he documentation on Integration Daemons there is also a[Postman Collection](https://egeria.odpi.org/open-metadata-resources/open-metadata-tutorials/postman-tutorial/) which contains examples of the commands needed to enable the running of different types of integration daemon.
+So this readme will only discuss the specific configuration needed to run an instance of the postgres connector, as illustrated in the following diagram.
+![alt text](https://encrypted-tbn0.gstatic.com/images?q=tbn:ANd9GcTrdVMpQA6q1dlb4x3iiDP0W5OG4d3UNZUCnw&usqp=CAU "possible integration daemon configuration")
+
+#Overview
+The Postgres Database Integration Daemon Connector works by periodically making a JDBC connection to a postgres databsase instance running on postgres server. It's no actually important which database accepts the connection as 
+this connector retrieves the applicable metadata for postgres system tables provided by postgres implementation of the [information schema.](https://www.postgresql.org/docs/9.1/information-schema.html)
+
+This Integration Daemon postgres connector supports the following types of database entity and will store the relevant metatdata for each corresponding egeria
+
+* Databases
+* Schemas
+* Tables
+* Views
+* Columns
+* Primary Keys
+* Foreign Keys 
+
+# Starting the Egeria Platform
+Below is an example of the java commandline used to launch the Egeria platform that will host the Integration Daemon that is in turn hosting the 
+postgres connector instance we are about to configure.
+The point of interest when launching a platform that will host connectors is the `-Dloader.path=/home/wbittles/egeria-database-connectors/postgres-connector/libs` command line option.
+The option specifies a file system directory that must contain both the postgres JDBC driver and the postgres-connector jar file.
+````
+java -Dloader.path=/home/wbittles/egeria-database-connectors/postgres-connector/libs -Dserver.port=9443 -jar /home/wbittles/egeria/open-metadata-distribution/open-metadata-assemblies/target/egeria-2.10-SNAPSHOT-distribution/egeria-omag-2.10-SNAPSHOT/server/server-chassis-spring-2.10-SNAPSHOT.jar`
+````
```

Comment:
If using the egeria docker image see https://github.com/odpi/egeria/tree/master/open-metadata-resources/open-metadata-deployment/docker/egeria 'Extending the image' which refers to how to add additional connectors.

If using the egeria-base helm chart see https://github.com/odpi/egeria/tree/master/open-metadata-resources/open-metadata-deployment/charts/egeria-base 'Using additional connectors'

(may be able to find better formatted doc consistently on egeria.odpi.org)

Reply:
I'll also update/write up some notes on how I've tested which may give another example